From 9acb289d1ac8b55b18590b99064bc56d24677b92 Mon Sep 17 00:00:00 2001
From: Wichert Akkerman <wichert@wiggy.net>
Date: Thu, 23 Oct 2014 09:59:28 +0200
Subject: [PATCH] Do not force -O2 compilation

This should be left to the configure script. This makes sane debugging of chilli possible by allowing compilation withit optimisation.
---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -26,7 +26,7 @@ iphash.c lookup.c system.h util.c option
 garden.c dns.c session.c pkt.c chksum.c net.c safe.c
 
 AM_CFLAGS = -D_GNU_SOURCE -Wall -Werror -fno-builtin -fno-strict-aliasing \
-  -O2 -fomit-frame-pointer -funroll-loops -pipe -I$(top_builddir)/bstring \
+  -fomit-frame-pointer -funroll-loops -pipe -I$(top_builddir)/bstring \
  -DDEFCHILLICONF='"$(sysconfdir)/chilli.conf"'\
  -DDEFPIDFILE='"$(localstatedir)/run/chilli.pid"'\
  -DDEFSTATEDIR='"$(localstatedir)/run"'\
